app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery
+  protect_from_forgery with: :null_session
```

Comment:
I've never seen just 'protect_from_forgery' without an argument. I wonder how that was working before? 

We'll need to do some testing today to make sure this configuration will work for the other api call .. looks like it will?!?